roxie/udplib/udptrs.cpp

```diff
@@ -394,6 +403,11 @@ class UdpReceiverEntry : public IUdpReceiverEntry
         CLeavableCriticalBlock block(activeCrit);
         if (maxRequestDeadTimeouts && (timeouts >= maxRequestDeadTimeouts))
         {
+            int timeExpired = msTick()-requestExpiryTime;
+            StringBuffer s;
+            EXCLOG(MCoperatorError,"ERROR: UdpSender: too many timeouts - aborting sends. Timed out %i times (flow=%u, max=%i, timeout=%u, expiryTime=%u[%u] ack(%u)) waiting ok_to_send for %u packets from node=%s",
```

Comment:
I know we [always] want to log this - but it if gets logged *a lot* we may need to adjust timeouts to prevent this.